test/fakeintake/client/client.go

```diff
@@ -94,11 +102,20 @@ func (c *Client) getLogs() error {
 }
 
 func (c *Client) getFakePayloads(endpoint string) (rawPayloads []api.Payload, err error) {
-	resp, err := http.Get(fmt.Sprintf("%s/fakeintake/payloads?endpoint=%s", c.fakeIntakeURL, endpoint))
-	if err != nil {
-		return nil, err
-	}
+	var resp *http.Response
+	for _, sleepTime := range retrySchedule {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Have you considered using `backoff` to handle the retry logic ?

Reply:
At the time I did it, we didn't had the dependency and I thought it better to avoid adding new dependencies. But since we're adding it back anyway with the PR to disable the LoadBalancer, I'll try to use it to see if it's cleaner.